kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -136,7 +145,8 @@ return {
     increment = function(conf, limits, identifier, current_timestamp, value)
       local red = redis:new()
       red:set_timeout(conf.redis_timeout)
-      local ok, err = red:connect(conf.redis_host, conf.redis_port)
+      local ok, err = red:connect(conf.redis_host, conf.redis_port,
+                                  { pool = get_redis_pool_name(conf) })
```

Comment:
We could reuse this table and just override its `pool` property, to save some GC overhead, and some function call overhead as well:
```lua
sock_opts.pool = get_redis_pool_name(conf) -- could even be inlined
```